### DISCUSSION TITLE
How to filter packages like yarn workspaces/Lerna

### DISCUSSION THREAD

**Discussion by nhattran998** [pnpm/pnpm#3460] [Q&A] [answered]
For example, I have a list of packages with the alias `@project/...`

And I want to filter N packages in M packages, how can I do that?
In Lerna, its really simple like `--scope=@project/{a,b,c}` --> Just select 3 packages in M packages . 

Hope anyone can help me, thank you <3!

**Comment by zkochan** [comment, marked as answer]
See this doc page: https://pnpm.io/filtering

**Reply by nhattran998** [reply]
Yes, I read this doc and I can not find the way to do that!
I just work around that with

```
pnpm --filter ./packages --filter !@project/web --filter !@project/mobile --filter !@project/abc --parallel run dev
```
Instead of doing like that, maybe just 
```
pnpm --filter ./packages --filter !@project/{web,mobile,abc} --parallel run dev
```
Hope you can clear that 😁

**Reply by zkochan** [reply]
pnpm currently only supports a simple pattern syntax using `*`, so `@project/{web,mobile,abc}` is currently not possible.

**Reply by nhattran998** [reply]
Thank you, guy! But I curiously why Pnpm not implement that?

**Reply by zkochan** [reply]
We try to keep it simple. There are many ways to solve your problem. Use some naming conventions or a different directory layout.
